pdns/dbdnsseckeeper.cc

```diff
@@ -74,11 +74,11 @@ bool DNSSECKeeper::isPresigned(const DNSName& name)
   return meta=="1";
 }
 
-bool DNSSECKeeper::addKey(const DNSName& name, bool keyOrZone, int algorithm, int bits, bool active)
+bool DNSSECKeeper::addKey(const DNSName& name, bool isSEP, int algorithm, int bits, bool active)
```

Comment:
`isSEP` is a confusing name - the way we use it, `hasSEPbit` might be cleaner (even if a bit long)